app/components/start_page_banner/view.rb

```diff
@@ -0,0 +1,2 @@
+class StartPageBanner::View < ViewComponent::Base
```

Comment:
any reason why these need to be namespaced?

Reply:
Good point, personally I feel it's much easier to spot what files are used for a particular component and it's easier to maintain rather than having a LONG list of components all lumped in together.

I'll remove the namespace to follow other components and increase the existing list of files in the components folder